plutus-tx-plugin/test/BuiltinList/Budget/9.6/indexNegative.uplc.golden

```diff
@@ -1,22 +1 @@
-(program
-   1.1.0
-   ((\traceError ->
-       (\go xs -> traceError "PT21")
-         ((\s -> s s)
-            (\s xs i ->
-               force
-                 (force (force chooseList)
-                    xs
-                    (delay (\ds -> traceError "PT22"))
-                    (delay
-                       ((\x xs ds ds ->
-                           force
-                             (force ifThenElse
-                                (equalsInteger 0 i)
-                                (delay x)
-                                (delay (s s xs (subtractInteger i 1)))))
-                          (force headList xs)
-                          (force tailList xs))))
-                 (constr 0 [])
-                 (constr 0 []))))
-      (\str -> (\x -> error) (force trace str (constr 0 [])))))
\ No newline at end of file
+(program 1.1.0 (\xs -> (\x -> error) (force trace "PT21" (constr 0 []))))
```

Comment:
Same here: the aspect you're trying to test (how are negative indexes handled) got optimised away.